docs/formatters/stylish/index.html

```diff
@@ -3,13 +3,13 @@
 description: Human-readable formatter which creates stylish messages.
 descriptionDetails: |-
 
-  The output matches that produced by eslint's stylish formatter. Its readability
-  enhanced through spacing and colouring
+  The output matches what is produced by eslint's stylish formatter. Its
+  readability is enhanced through spacing and colouring.
```

Comment:
this file is not meant to be manually edited. the content are generated from the metadata in the source file:
https://github.com/palantir/tslint/blob/master/src/formatters/stylishFormatter.ts#L32

Reply:
Thanks. I pushed a new commit.